src/main/java/racingcar/domain/CarName.java

```diff
@@ -9,18 +10,20 @@
     private final String name;
 
     public CarName(final String name) {
-        checkName(name);
-        this.name = name;
+        final String noneNullString = Optional.ofNullable(name)
+                .orElseThrow(() -> new IllegalArgumentException("null은 사용할 수 없습니다. String 타입을 사용하세요."));
+        checkName(noneNullString);
+        this.name = noneNullString;
     }
 
     private static void checkName(final String name) {
-        checkNullOrBlank(name);
+        checkBlank(name);
         checkMinimumCarName(name);
         checkMaximumCarNameLength(name);
     }
 
-    private static void checkNullOrBlank(String name) {
-        if (name == null || name.isBlank()) {
+    private static void checkBlank(String name) {
+        if (name.isBlank()) {
```

Comment:
검증로직을 static으로 잡은 이유가 있을까요?

Reply:
```suggestion
    private static void checkBlank(String name) {
        if (Objects.isNull(name) || name.isBlank()) {
```
추가로 방어적으로 null 체크를 진행하면 좋겠네요.

Reply:
리펙토링 과정에서 깜빡하고 수정하지 못했던 부분이네요..